src/languages/en_GB.json

```diff
@@ -227,6 +227,8 @@
     "setLang": "Set Language to",
     "showUnits": "Show units",
     "showEmblems": "Show emblems",
-    "setName": "Click to change your display name"
+    "setName": "Click to change your display name",
+    "toggleDarkMode": "Toggle dark mode",
```

Comment:
Is this used anywhere? :thinking:

Reply:
That is a relic of my first implementation with just two modes. I will remove it.